docs/tutorial/advanced.rst

```diff
@@ -0,0 +1,20 @@
+Advanced Capabilities and Features
+======================================
+
+More like these are all the things we can do with this, but will not be showing
+
+- mention parameter tuning
+- say that parameters are not preset and need to be tuned for each dataset
+
+CHTC integration
```

Comment:
CHTC is local to our university. The way to say it may be Snakemake integration with cloud and high-throughput computing resources, which we've prototyped in our local cluster. If we start testing in OSG that would be different because many people are eligible for accounts.